docs/conf.py

```diff
@@ -25,7 +25,7 @@
 # -- Project information -----------------------------------------------------
 
 project = py_pkg.__name__
-copyright = "2006 - 2023, Mathieu Fenniak and pypdf contributors"
+copyright = "2006 - 2024, Mathieu Fenniak and pypdf contributors"
```

Comment:
I recommend generating this on-the-fly instead.

Reply:
Good idea did not know this was possible.